### PR TITLE
integration: expose ElectionTimeout method

### PR DESCRIPTION
**Pull request by gyuho**
Addressing https://github.com/coreos/etcd/issues/8781.

### CODE REVIEW

File: etcdserver/config.go [hunk 1]
```diff
@@ -176,7 +176,7 @@ func (c *ServerConfig) ReqTimeout() time.Duration {
 	return 5*time.Second + 2*time.Duration(c.ElectionTicks*int(c.TickMs))*time.Millisecond
 }
 
-func (c *ServerConfig) electionTimeout() time.Duration {
+func (c *ServerConfig) ElectionTimeout() time.Duration {
```
Comment (xiang90) [inline etcdserver/config.go]: no need to expose this, right? exposing the one in cluster.go is good enough

Comment (gyuho) [reply etcdserver/config.go]: @xiang90 Another question. Do we want to multiply election ticks by heartbeat ticks (as `etcdserver` does`) for `integration` package?

Comment (xiang90) [reply etcdserver/config.go]: i am not sure what you mean here. can you provide me the link to the code you referred to ? thanks!

Comment (gyuho) [reply etcdserver/config.go]: https://github.com/coreos/etcd/blob/87ad10c1552f163f541e89786597db888af8e1a6/etcdserver/config.go#L179-L181

vs

https://github.com/coreos/etcd/blob/87ad10c1552f163f541e89786597db888af8e1a6/integration/cluster.go#L611-L613

Comment (xiang90) [reply etcdserver/config.go]: the second one seems to be wrong.

**Comment by gyuho** [comment]
@xiang90 PTAL.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**